src/test/java/model/input/file/XlsxFileTest.java

```diff
@@ -13,17 +10,14 @@
 import model.data.DataColumn;
 import model.data.DataRow;
 import model.data.DataTable;
-import model.data.value.DataValue;
-import model.data.value.FloatValue;
-import model.data.value.IntValue;
-import model.data.value.StringValue;
+import model.data.value.*;
 
 import org.junit.Before;
 import org.junit.Test;
 
 /**
  * JUnit test for the XlsFile class.
- * @author pablo
+ * @author Paul
```

Comment:
De username van mijn OS is pablo. Ik had daar een auto generated javadoc over het hoofd gezien :P